pages.zh/osx/afinfo.md

```diff
@@ -0,0 +1,24 @@
+# afinfo
+
+> 显示音频文件元数据 (Metadata) 详细信息 (OS X)
```

Comment:
```suggestion
> 显示音频文件元数据 (Metadata) 详细信息 (OS X).
```

We usually add a `.` at the end of the line. Although it is not the Chinese period.